oioioi/programs/templates/programs/report-comments.html

```diff
@@ -24,7 +24,22 @@
                     {% endif %}
                 {% endif %}
                 {% if allow_test_comments %}
-                    <span>{{ test.comment }}</span>
+                    <span>
+                    {% if test.comment %}
+                        {% if test.result_percentage %}
+                            {% blocktranslate with test.result_percentage as percentage %}
+                                Granted {{ percentage  }}% points, comment:
```

Comment:
How about "Granted {{ percentage }}% *of* points". Also as `percentage` could be 33.33333333 I think we should pretty print this floating point value